package sofa.pbrpc.test;

option cc_generic_services = true;
option java_generic_services = true;

message EchoRequest {
    required string message = 1;
}

message EchoResponse {
    required string message = 1;
}

service EchoServer {
    rpc TestEcho(EchoRequest) returns(EchoResponse);
}
